package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gaia-button",
+  "version": "0.0.5",
+  "homepage": "https://github.com/gaia-components/gaia-button",
+  "license": "MIT",
+  "main": "gaia-button.js",
+  "devDependencies": {
```

Comment:
I did not seen the `dependencies` in `gaia-header` https://github.com/gaia-components/gaia-header/blob/master/package.json , is it not handled in `bower.json`?